scripts/update-version.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const fs = require('fs');
+
+module.exports = function updateCoreVersion(newVer) {
```

Comment:
This function should be made to be async and awaited in `scripts/release-branch.js`:
```suggestion
module.exports = async function updateCoreVersion(newVer) {
```